lib/poll.ml

```diff
@@ -11,7 +11,13 @@ module Raw = struct
 end
 
 module Flags = struct
-  type t = int
+  type output = < output: unit; >
+
+  type 'a input = < .. > as 'a
```

Comment:
I dislike that the definitions are not symetric, it also feels weird why input would have a constructor, as an innocent user reading this for the first time it would lead me into the wrong direction